include/recorded_battle.h

```diff
@@ -13,7 +14,7 @@ struct RecordedBattleSave
     u8 playersGender[MAX_BATTLERS_COUNT];
     u32 playersTrainerId[MAX_BATTLERS_COUNT];
     u8 playersLanguage[MAX_BATTLERS_COUNT];
-    u32 rngSeed;
+    rng_value_t rngSeed;
```

Comment:
It absolutely does break saves. I guess in theory I could add some code to check if the recorded battle save area size is wrong and only delete that?

Reply:
Maybe another maintainer wants to come along and say otherwise, but personally I wouldn't worry about the recorded battles at all.

I expect that Expansion should probably do something like embed the current version number in there somewhere, and then refuse to replay battles from a previous version, which is totally out of scope for this change. There's no way we'd be able to guarantee that `Random()` calls always occur in the same order, which is all-but-guaranteed to effectively break the recorded battle, so it seems silly to make you change anything on this PR when other PRs down the line are likely to break it anyway :)